atomic_reactor/core.py

```diff
@@ -557,3 +557,11 @@ def image_exists(self, image_id):
             response = response is not None
         logger.debug("image exists: %s", response)
         return response
+
+    def get_info(self):
+        """
+        get info about used docker environment
+
+        :return: tuple of (`docker info`, `docker version`)
```

Comment:
Wouldn't `get_info()` and `get_version()`, each returning one of the parts of that tuple, be less confusing?